### DISCUSSION TITLE
Python SDK Documentation

### DISCUSSION THREAD

**Discussion by YannickLecroart21** [argoproj/argo-workflows#7362] [Q&A] [answered]
Hi,

I would like to suspend or resume a Cron Workflow using the Python SDK but I cannot find any documentation on this subject. 

I tried to use the following procedure but it has been deprecated and no longer work.

`https://github.com/argoproj-labs/argo-client-python/blob/master/docs/CronWorkflowServiceApi.md#updateCronWorkflow`

Could you please help me out?

Thanks in advance. 

Regards

**Comment by terrytangyuan** [comment, marked as answer]
Python SDK is currently being re-worked on in https://github.com/argoproj/argo-workflows/tree/master/sdks/python. Stay tuned!

**Reply by YannickLecroart21** [reply]
Thanks for your quick reply. Can I try it already?

**Reply by terrytangyuan** [reply]
We are still working on some last-mile issues as well as releasing it to PyPI. I'd suggest trying it out once it's on PyPI.
